src/js/bootstrap-datetimepicker.js

```diff
@@ -925,6 +1153,11 @@
                 widget.off('click', '[data-action]');
                 widget.off('mousedown', false);
 
+                widget.find('[tabindex="0"]').off({
+                    'keydown': keydown,
+                    'keyup': keyup
+                });
```

Comment:
Yes, among others. (I'm assuming you are talking about the events on `[tabindex="0"]` elements in general, and not the removing of events in the `hide` function, right?). If you have toolbars enabled (if `sideBySide: false`, `showTodayButton: true`, `showClear: true` or `showClose: true`), you can `tab` to the different buttons as well, giving you different context, which the screen reader will tell you about.